os/board/rtl8720e/src/component/wifi/wifi_interactive_mode.c

```diff
@@ -793,6 +795,7 @@ static int _netlib_setmacaddr(const char *ifname, const uint8_t *macaddr)
 
 			/* Put the driver name into the request */
 			strncpy(req.ifr_name, ifname, 6);

```

Comment:
To reduce duplicated operation, how about `5` instead of `6`? Because last index is set as `\0` at line 798 always.

Reply:
@sunghan-chang , how about modifying like below

_strncpy(req.ifr_name, ifname, IFNAMSIZ - 1);
req.ifr_name[IFNAMSIZ - 1] = '\0';_